app/javascript/components/patient/assessment/AssessmentTable.js

```diff
@@ -292,9 +291,7 @@ class AssessmentTable extends React.Component {
             <span className="ml-2">Edit</span>
           </Dropdown.Item>
           <AddAssessmentNote assessment={rowData} patient={this.props.patient} authenticity_token={this.props.authenticity_token} />
-          {!this.props.patient.isolation && (
-            <ClearSingleAssessment assessment_id={rowData.id} patient={this.props.patient} authenticity_token={this.props.authenticity_token} />
-          )}
+          <ClearSingleAssessment assessment_id={rowData.id} patient={this.props.patient} authenticity_token={this.props.authenticity_token} />
```

Comment:
This modal (and `ClearAssessments`) should have an additional option to turn on Asymptomatic, per slide 5 in the guidance here:

![image](https://user-images.githubusercontent.com/8235974/117191356-77c13e80-adae-11eb-97d2-3bf89e359ca0.png)

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/861/commits/94f200ed1eb441b2232d281db89e9f25a51209ad#diff-979b65d9207c528702a696244f7be7aad5dc37fe6bc98a8eca1d10d4bb19995aR115